### PR TITLE
[i10n] Update translations to Spanish

### PR DESCRIPTION
**Pull request by elima**
Currently translated at 100.0% (594 of 594 strings) \o/. 

Thanks, @jaragunde.

### CODE REVIEW

File: app/src/main/res/values-es-rES/strings.xml [hunk 1]
```diff
@@ -573,1092 +445,795 @@
...
     <string name="privacy_options_saved_logins_clear_all">Borrar todo</string>
-
     <!-- This string labels the Delete button from the Login edit dialog deletes the current login. -->
     <string name="privacy_options_saved_login_delete">Eliminar</string>
-
     <!-- This string is used to label radio buttons for setting ETP in.
          '%1$' Will be replaced at runtime with the Learn More url for ETP. -->
-    <string name="privacy_options_tracking"><![CDATA[Protección mejorada contra el rastreo. (<a href="%1$s">Saber más</a>)]]></string>
-
+    <string name="privacy_options_tracking">Protección mejorada contra el rastreo. (<a href="%1$s">Saber más</a>)</string>
```
Comment (jaragunde) [inline app/src/main/res/values-es-rES/strings.xml:1021]: I hope this string works correctly! In weblate, I received this as a string with modifications, and the commas in the href were missing.

**Review by jaragunde — COMMENTED** [review]
Thanks Edu! Maybe I'm being extra-cautious, but it might be worth testing one changed string, see comment.

File: app/src/main/res/values-es-rES/strings.xml (hunk 1 above)
Comment (elima) [reply app/src/main/res/values-es-rES/strings.xml:1021]: Ok, I will build it and test it myself and let you know. Thanks!

**Review by felipeerias — APPROVED** [review]
LGTM

File: app/src/main/res/values-es-rES/strings.xml (hunk 1 above)
Comment (elima) [reply app/src/main/res/values-es-rES/strings.xml]: @jaragunde it turns out this was a problem. Without the CDATA block, the link is removed. The text `Saber mas` is there, but without active link.

I added a new commit putting the CDATA block back. I'm afraid I will have to do this manually for now, for all translations that contain embedded HTML :/

Good catch!

**Pull request merged**